### PR TITLE
Change project structure to be more modular.

### PR DESCRIPTION
**Pull request by clpatterson**
I modified the api project structure to follow suggested approach in [Flask-Restful documentation](https://flask-restful.readthedocs.io/en/latest/intermediate-usage.html).
```
myapi/
    __init__.py
    app.py          # this file contains your app and routes
    resources/
        __init__.py
        foo.py      # contains logic for /Foo
        bar.py      # contains logic for /Bar
    common/
        __init__.py
        util.py     # just some common infrastructure
```

### CODE REVIEW

**Comment by jennyxing** [comment]
is `__init__.py` meant to be empty? I'm seeing two instances of it, one at the root and one in `/resources`

File: api.py [hunk 1]
```diff
@@ -1,114 +1,12 @@
-from flask import Flask, jsonify, abort, url_for
+from flask import Flask, abort
 from flask_restful import Api, Resource, reqparse, fields, marshal
+from resources.reservations import ReservationList, Reservation
```
Comment (jennyxing) [inline api.py:3]: I'm getting an error on this line:
`ModuleNotFoundError: No module named 'resources'`

Comment (clpatterson) [reply api.py:3]: When you run api.py from which directory are you running it? I can get it to work from hotel_api/...but I think I'm missing some best practices here in the way I have it currently configured.

**Comment by clpatterson** [comment]
__init__.py can be empty and it is technically not required in python3, but maybe something else in the init is required.

**Pull request merged**